Week2/HomeWork/app.js

```diff
@@ -0,0 +1,123 @@
+const mysql = require('mysql');
+const prompt = require('prompt');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const capitalCity = `select name from city where id in (select capital from country where name = ?)`;
+
+const regions = `select distinct language from countrylanguage where coutrycode in (select code from county where region = ? )`;
+
+const cities = `select count(distinct name) as 'total' from city where countrycode in (select from countrylanguage where language = ?)`;
+
+const officialLanguage = `select name from country where region = ? 
+    and code in(select countrycode from countrylanguage where language = ? and isofficial = 't')`;
+
+const languages = `select country.continent, count(distinct language) as 'total language' from country
```

Comment:
be careful you have used county instead of country again :)

Reply:
thank you for review and feedback .
issues are fixed :)